lib/ansible/module_utils/basic.py

```diff
@@ -2309,6 +2309,13 @@ def exit_json(self, **kwargs):
         self._return_formatted(kwargs)
         sys.exit(0)
 
+    def fail_dependency(self, msg):
```

Comment:
instead of fail_json we might want to separate message construction to be able to use from other points, like existing fail_json or an exception hook.